federatedscope/autotune/baseline/fedex_vfl.yaml

```diff
@@ -0,0 +1,53 @@
+use_gpu: False
```

Comment:
FedEx is designed for horizontal scenarios, imo. When clients have totally different inputs, I am wondering whether updating a policy based on all their feedback is still a reasonable trick. I suggest you (and @qbc2016 ?) to have a glance at <FLoRA>.